src/validation.h

```diff
@@ -185,6 +184,16 @@ struct MempoolAcceptResult {
     /** Raw base fees in satoshis. */
     const std::optional<CAmount> m_base_fees;
 
+    static MempoolAcceptResult Failure(TxValidationState state) {
+        return MempoolAcceptResult(state);
+    }
+
+    static MempoolAcceptResult Success(std::list<CTransactionRef>&& replaced_txns, CAmount fees) {
```

Comment:
In 0d868d0abbc9655e2e70766433d53719debf27b4 "[validation] package validation for test accepts"

nit: I would prefer for these changes to `MempoolAcceptResult` be split into a previous commit. It would make this commit slightly easier to read IMO.

Reply:
Done